app/concepts/matestack/ui/core/collection/filter/input/input.haml

```diff
@@ -0,0 +1,8 @@
+- if [:text, :number, :email, :date, :password].include?(options[:type])
+  %input{"v-model#{'.number' if options[:type] == :number}": input_key,
```

Comment:
What does this comment here do?

Reply:
this is not a comment, it's just wrong syntax highlighting of github ;)